server/src/main/java/io/spine/quickstart/server/ServerApp.java

```diff
@@ -120,17 +116,7 @@ public static void main(String[] args) throws IOException {
         container.awaitTermination();
     }
 
-    /**
-     * A singleton logger to use in scope of this application.
-     */
-    private enum LogSingleton {
-        INSTANCE;
-
-        @SuppressWarnings("NonSerializableFieldInSerializableClass")
-        private final Logger value = LoggerFactory.getLogger(ServerApp.class);
-    }
-
     private static Logger log() {
-        return LogSingleton.INSTANCE.value;
+        return Logging.get(ServerApp.class);
```

Comment:
Can we please move away from a static method, but use `extends Logging` feature.

Reply:
Let's discuss.